### PR TITLE
new command join and remove to force join or remove avatars into your current team

### PR DESCRIPTION
**Pull request by lsCoding666**
new command join and remove to force join or remove avatars into your current team
@Command(label = "join", usage = "join <avatar id> <avatar id> <avatar id>...",
        description = "force join avatar into your team", permission = "player.join")

@Command(label = "remove", usage = "remove <index> <index> <index>...",
        description = "force remove avatar into your team", permission = "player.remove")

### CODE REVIEW

File: src/main/java/emu/grasscutter/Config.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public static class GameServerOptions {
 		public int InventoryLimitMaterial = 2000;
 		public int InventoryLimitFurniture = 2000;
 		public int InventoryLimitAll = 30000;
-		public int MaxAvatarsInTeam = 4;
+		public int MaxAvatarsInTeam = 9;
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/Config.java]: Probably shouldn't be included

Comment (lsCoding666) [reply src/main/java/emu/grasscutter/Config.java]: image

Maybe we shoud not limit MaxAvatarsInTeam on server. Client has some limit like 5. If you force add avatar into your team, the maxium is 5. When your team has 5, force join can not has response.

Comment (lsCoding666) [reply src/main/java/emu/grasscutter/Config.java]: video:  [click](https://lsgdut-my.sharepoint.com/:v:/g/personal/lsgdut_c34_pw/EcDJMmtPjkFNnL2O4GI4hAoBPn_0hoiVLTmRj90z1T9wcg?e=VkxPK1)

Comment (ghost) [inline src/main/java/emu/grasscutter/Config.java]: Please keep this the default value of 4.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/Config.java (hunk 1 above)
Comment (lsCoding666) [reply src/main/java/emu/grasscutter/Config.java]: okay，changed

**Comment by ghost** [comment]
Please fix your merge conflicts so I can merge this PR.

**Comment by lsCoding666** [comment]
> Please fix your merge conflicts so I can merge this PR.

Sorry to see it so late. I'll change it right away

**Comment by lsCoding666** [comment]
> Please fix your merge conflicts so I can merge this PR.

Sorry to see it so late. I'll change it right away

**Comment by lsCoding666** [comment]
> Please fix your merge conflicts so I can merge this PR.

merged.@Melledy

**Review by ghost — APPROVED** [review]

**Pull request merged**